docs/src/main/asciidoc/azure-functions-http.adoc

```diff
@@ -50,68 +71,92 @@ If you don't log in to Azure you won't be able to deploy.
 az login
 ----
 
-== Quarkus dev mode
+== Run your Quarkus application in dev mode
 
-Quarkus dev mode works by just running your application as a HTTP endpoint.
+In Quarkus dev mode, your application just runs as an HTTP endpoint without integration with Azure Functions.
 
 [source,bash,subs=attributes+]
 ----
 ./mvnw clean package quarkus:dev
 ----
```

Comment:
Replace this w/ include of dev mode snippet?

```
include::{includes}/devtools/dev.adoc[]
```

Reply:
Fixed.